cli/compose/loader/loader.go

```diff
@@ -62,14 +85,18 @@ func Load(configDetails types.ConfigDetails) (*types.Config, error) {
 			return nil, err
 		}
 
-		var err error
-		configDict, err = interpolateConfig(configDict, configDetails.LookupEnv)
-		if err != nil {
-			return nil, err
+		if !opts.SkipInterpolation {
+			var err error
```

Comment:
Can you remove this `var err ..` ? There's more `err` variables in this function, so having this here will be confusing.